src/components/dictionaryConcepts/components/ConceptNameRows.jsx

```diff
@@ -20,7 +20,6 @@ class ConceptNameRows extends Component {
       id: '',
       name: '',
       locale: 'en',
-      locale_full: { value: 'en', label: 'English [en]' },
```

Comment:
How did the above line cause the error?

Reply:
The line did not cause the error but was redundant as the locale_full is fetched using locale.
the error was caused by.

The error was caused by line 22  of
src/components/dictionaryConcepts/components/CreateConceptTable.jsx
where `newRow={{ uuid: newRow }}` over wrote the newRole in default props

Reply:
The changes in this pull request should be only and only those required to fix the error as reported on the ticket. If you wanna do some clean up, that should be a separate ticket and hence pull request.

Reply:
okay, will rectify that.